appcues/src/main/java/com/appcues/ui/utils/ImageLoaderWrapper.kt

```diff
@@ -0,0 +1,24 @@
+package com.appcues.ui.utils
+
+import android.content.Context
+import android.os.Build.VERSION
+import android.os.Build.VERSION_CODES
+import coil.ImageLoader
+import coil.decode.GifDecoder
+import coil.decode.ImageDecoderDecoder
+import coil.decode.SvgDecoder
+
+internal class ImageLoaderWrapper(private val context: Context) {
```

Comment:
we may want to filter out *Wrapper.kt" classes like this one, ContextWrapper, RetrofitWrapper

Reply:
i'd vote no as we chatted, but i guess no big deal. It should be a tiny number